### ISSUE TITLE
Loss is about 2.x, but mAP is not good, only about 66.x%.

### ISSUE THREAD

**Issue by XiangqianMa** [amdegroot/ssd.pytorch#331]
I have tried many training tricks, such as different batch_size, different learning rate. But, almost each loss is 2.x. I checked other issues, their mAP are 7x.x%, but mine is 66.x%. So does anyone encounter such problem?

**Comment by yyx1107** [comment]
Have you resolved it?

**Comment by yinyuhangi** [comment]
My loss is also 2.x, mAP is 68.3%

**Comment by rechealzrq** [comment]
Have you solved your problem?

**Comment by ChenT1994** [comment]
> I have tried many training tricks, such as different batch_size, different learning rate. But, almost each loss is 2.x. I checked other issues, their mAP are 7x.x%, but mine is 66.x%. So does anyone encounter such problem?



> I have tried many training tricks, such as different batch_size, different learning rate. But, almost each loss is 2.x. I checked other issues, their mAP are 7x.x%, but mine is 66.x%. So does anyone encounter such problem?



> I have tried many training tricks, such as different batch_size, different learning rate. But, almost each loss is 2.x. I checked other issues, their mAP are 7x.x%, but mine is 66.x%. So does anyone encounter such problem?

I have the same problem,Did you find the reason?

**Comment by Bobby2090** [comment]
> 
> 
> I have tried many training tricks, such as different batch_size, different learning rate. But, almost each loss is 2.x. I checked other issues, their mAP are 7x.x%, but mine is 66.x%. So does anyone encounter such problem?

我最近也遇到了这个问题，你解决了吗？求指导，谢谢
